
e.org.altemista.cloudfwk.common.cryptography.exception.no_keystore=The keystore information is required
e.org.altemista.cloudfwk.common.cryptography.exception.no_keystore_resource=The resource location of the keystore is required
e.org.altemista.cloudfwk.common.cryptography.exception.no_keystore_password=The password of the keystore is required
e.org.altemista.cloudfwk.common.cryptography.exception.no_keystore_type=The type of the keystore is required

e.org.altemista.cloudfwk.common.cryptography.exception.invalid_keystore_type=Keystore type {} not available
e.org.altemista.cloudfwk.common.cryptography.exception.invalid_keystore_algorithm=The algorithm used to check the integrity of the keystore cannot be found
e.org.altemista.cloudfwk.common.cryptography.exception.invalid_keystore_certificate=Any of the certificates in the keystore could not be loaded
e.org.altemista.cloudfwk.common.cryptography.exception.io_exception=I/O or format problem with the keystore data

e.org.altemista.cloudfwk.common.cryptography.exception.invalid_keymanager_algorithm=Algorithm {} not available
e.org.altemista.cloudfwk.common.cryptography.exception.invalid_keymanager_keystore=Operation failed
e.org.altemista.cloudfwk.common.cryptography.exception.invalid_keymanager_key=A key could not be recovered

e.org.altemista.cloudfwk.common.cryptography.exception.invalid_trustmanager_algorithm=Algorithm {} not available
e.org.altemista.cloudfwk.common.cryptography.exception.invalid_trustmanager_keystore=Operation failed

e.org.altemista.cloudfwk.common.cryptography.exception.publickey_invalid_keystore=Keystore at {} uninitialized
e.org.altemista.cloudfwk.common.cryptography.exception.publickey_notfound=No public key with alias {} found in the keystore at {}

e.org.altemista.cloudfwk.common.cryptography.exception.privatekey_algorithm=The algorithm for recovering the key cannot be found
e.org.altemista.cloudfwk.common.cryptography.exception.privatekey_invalid_key=The key could not be recovered
e.org.altemista.cloudfwk.common.cryptography.exception.privatekey_invalid_keystore=Keystore at {} uninitialized
e.org.altemista.cloudfwk.common.cryptography.exception.privatekey_notfound=No private key with alias {} found in the keystore at {}
